content/_guides/channelmodes.md

```diff
@@ -4,25 +4,234 @@ category: chanop
+Channel modes and `ChanServ` flags are tools for configuring and managing
+channels.
 
-All channel modes will be lost when a channel becomes empty. Enable GUARD to
-preserve modes.
+Channel modes are the most basic form of channel configuration. Modes can be
+set using the IRC `/mode` command or, for some modes, via `ChanServ`.
 
-To set a mode, use `/mode #channel +(mode)` replacing `(mode)` with the letter
-that corresponds to the mode. To unset a mode, use `/mode #channel -(mode)`
+The `ChanServ` channel service provides additional administration functions
+for registered channels. It is complementary to modes. You can perform many of
+the same functions manually, but using `ChanServ` to administer your channel
+where applicable offers some benefits:
 
-## Available channel modes
+- It is always opped, so it can set and enforce IRC [channel
+  modes](#channel-modes) on your behalf.
+- It offers granular privileges through [user flags](#chanserv-user-flags),
+  allowing you to delegate limited capabilities to other users. 
+- It can maintain channel attributes and perform functions automatically
+  through [control flags](#chanserv-control-flags).
 
+## Channel modes
+
+Channel modes control:
+
+- Bans (`b`, `e`)
+- Moderation (`m`, `n`, `q`, `u`, `v`, `z`)
+- Forwarding (`f`, `F`)
+- Channel visibility (`p`, `s`)
+- Channel invitations (`g`, `i`, `I`)
+- Channel access controls (`j`, `k`, `l`, `n`, `o`, `Q`, `r`, `S`)
+
+### Requirements
+
+- To use `/mode`, you must be a channel operator (`+o`).
+- To use `ChanServ`, your channel must be
+  [registered](/creatingchannels#registering-the-new-channel) and your user
+  flags must allow you to take actions that set modes.
+
+### Mode command summary
+
+| Purpose                    |Command                    |
+|----------------------------|---------------------------|
+| List current channel modes | `/mode <channel>`         |
+| Set a mode                 | `/mode <channel> +<mode>` |
+| Unset a mode               | `/mode <channel> -<mode>` |
+
+### Examples
+
+To allow only operators to set the topic on `#foo`:
+
+```irc
+/mode #foo +t
+```
+
+To set a user `bar` as an operator on `#foo`:
+
+```irc
+/mode #foo +o bar
+```
+
+To ban any user connecting to `#foo` from a given host:
+
+```irc
+/mode #foo +b *!*@10-10-32-1.res.example.com
+```
+
+### Preserving channel modes
+
+By default, all channel modes are lost when a channel becomes empty. To
+preserve modes, set the `ChanServ` `GUARD` [control
+flag](#chanserv-control-flags): 
+
+```irc
+/msg ChanServ SET <channel> GUARD ON
+```
+
+`ChanServ` will join your channel and remain there, which helps prevent the
+channel from being empty, which helps prevent the modes from being lost.
+
+In addition, to prevent modes being lost even if `ChanServ` were to go offline
+and leave the channel empty, set the modes with `MLOCK` as described
+[below](#enforcing-channel-modes).
```

Comment:
"click here" or "see below" isn't great imo, rather try to get the title of the section into the sentence and make that the link, or "see enforcing channel modes"